changelog/v1.1.0/envoy-command-line-args.yaml

```
@@ -0,0 +1,7 @@
+changelog:
+- type: NEW_FEATURE
+  description: Add `extraEnvoyArgs` to .Values.gatewayProxies.NAME so users can add additional envoy command line arguments (e.g. --set component-log-level="upstream:trace,misc:error", to set log levels for individual envoy components). See the full list of Envoy command line args here - https://www.envoyproxy.io/docs/envoy/latest/operations/cli
```

```suggestion
  description: Add the Helm field `.Values.gatewayProxies.gatewayProxy.extraEnvoyArgs` so users can add additional envoy command line arguments (e.g. --set component-log-level="...", to set log levels for individual envoy components). See the full list of Envoy command line args here - https://www.envoyproxy.io/docs/envoy/latest/operations/cli
```
For both these duplicated lines: probably fine to replace NAME with `gatewayProxy`, as that's what most people will be using. And just a couple small wording changes.